### PR TITLE
Compute statistical values of data in real-time

### PR DESCRIPTION
**Pull request by kvedala**
#### Description of Change

# Whats new
Compute statistical values of data in real-time without *a priori* information of the total number of data samples. These include:
* mean
* variance
* standard deviation

# Fixes
* Added Google's style to `.vscode` file for auto-lint correction.
* fixed CPPLINT issues with `jump_search.cpp`
* ~~removed out of place object files added in 9f8aef5 (#569).~~ (fixed by @cclauss in #master)
* ~~removed out of place file paths added in 9f8aef5 (#569)~~ (fixed by @cclauss in #master)

#### Checklist

- [x] Added description of change
- [x] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTION.md#New-File-Name-guidelines)
- [ ] Added tests and example, test must pass
- [x] Relevant documentation/comments is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTION.md#Commit-Guidelines)
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] Sort by alphabetical order
- [x] I acknowledge that all my contributions will be made under the project's license.

Notes:

### CODE REVIEW

**Comment by kvedala** [comment]
The pull request is based from commit #601 because hte commit made after it - #569  has known issues:
1. cpplint check fails
2. includes object files in the source code

**Comment by cclauss** [comment]
Please rebase on master.  I have lifted and changed the `vscode` changes and pushed them to master (they should be quite helpful).  And fixed the #569 issues by hand.

**Comment by kvedala** [comment]
> Please rebase on master. I have lifted and changed the `vscode` changes and pushed them to master (they should be quite helpful). And fixed the #569 issues by hand.

Thank you @cclauss

File: .vscode/settings.json [hunk 1]
```diff
@@ -68,4 +68,4 @@
      "editor.formatOnSave": true,
      "editor.formatOnType": true,
      "editor.formatOnPaste": true
-}
+}
```
Comment (cclauss) [inline .vscode/settings.json:71]: Please add a carriage return at the end of the file.

**Comment by kvedala** [comment]
included in PR #805

**Pull request closed without merging**